examples/save_data_example.py

```diff
@@ -65,28 +71,28 @@ def run_scans(nScans, power_idx, field_idx, DNP_data=None):
 
 with power_control() as p:
     for j,this_dB in enumerate(dB_settings):
+        print("I'm going to pretend to run",this_dB,"dBm")
         if j == 0:
-            short_delay
+            time.sleep(short_delay)
             p.start_log()
         p.set_power(this_dB)
         for k in range(10):
-            short_delay
+            time.sleep(short_delay)
             if p.get_power_setting() >= this_dB: break
-        long_delay
+        time.sleep(long_delay)
         meter_powers[j] = p.get_power_setting()
         if True:
+            # the following seems unrealistic for a field sweep -- what's
+            # up with that?
             for B0_index,desired_B0 in enumerate(r_[3501:3530:0.001]):
```

Comment:
what's up here?  I can't believe you were really trying to run an experiment where you used 29,000 field sweep steps.

Reply:
I am exaggerating the length to show that for HUGE amounts it will still work